README.md

```diff
@@ -115,15 +108,63 @@ const { statusCode, data } = await anvilClient.fillPDF(pdfTemplateID, payload, o
   * `errors` (Array of Objects) - Will be present if status >= 400. See Errors
     * `message` (String)
 
+##### createEtchPacket(variables[, responseQuery])
+
+Creates an Etch Packet and optionally sends it to the first signer. See the [API Documentation](#api-documentation) area for details. See [Examples](#examples) area for examples.
+
+### Class Methods
+
+##### prepareGraphQLStream(pathOrStream[, options])
```

Comment:
I think we should pick one of these to be The Way to interact with the API. I would say maybe we change this to `prepareGraphQLFile` then do some checking: if buffer, call the buffer func, if stream || string, call the stream func. 

We can probably not document the base64 way in this client. The base64 way is really for people who arent using the client, e.g. someone in rubyland for now. It's the easiest way to interact with the raw API rather than having to deal with the upload munging like you have done here.

Reply:
OK, I'm down for this...